packages/fx-core/src/plugins/resource/spfx/plugin.ts

```diff
@@ -107,7 +107,7 @@ export class SPFxPluginImpl {
 
       await progressHandler?.next(ScaffoldProgressMessage.ScaffoldProject);
       const framework = ctx.answers![SPFXQuestionNames.framework_type] as string;
-      const solutionName = ctx.projectSettings?.appName as string;
+      const solutionName = ctx.projectSettings?.appName as string; // I did not change this.
```

Comment:
If you don't want to dismiss all the existing approvals, I'll remove the comment later in next PR.

Reply:
Thanks for catching this! I will have another PR to remove this comment after checking this one in :)

Reply:
Fixed